vita3k/gui/src/app_context_menu.cpp

```diff
@@ -307,7 +307,8 @@ void draw_app_context_menu(GuiState &gui, EmuEnvState &emuenv, const std::string
     auto &common = emuenv.common_dialog.lang.common;
     auto &lang_compat = gui.lang.compatibility;
 
-    const auto is_commercial_app = title_id.find("PCS") != std::string::npos;
+    const auto is_commercial_app = title_id.starts_with("PCS") || title_id == "NPXS10007";
+    const auto is_system_app = title_id.starts_with("NPXS") && title_id != "NPXS10007";
```

Comment:
```suggestion
    const auto is_commercial_app = title_id.starts_with("PCS") || (title_id == "NPXS10007");
    const auto is_system_app = title_id.starts_with("NPXS") && (title_id != "NPXS10007");
```

very small change ^^
is formating code